src/components/RemoveMovieBtn.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import '../style/BtnDesign.css'
+
+
+export default class RemoveMovieBtn extends Component {
```

Comment:
good extraction.
a better one will be to have a 1 button component that gets onclick event and button icon.

Reply:
Exactly, that's what i wanted to talk to you on Wednesday. How to implement it.